kotlin-format.hcl

```diff
@@ -0,0 +1,23 @@
+description = "A command-line tool designed to enforce consistent code formatting for Kotlin."
```

Comment:
- [ ] You'll want the runtime-dependencies thing here too (see ktfmt.hcl) since the script executes java and without the runtime-dependencies it may run some random broken version of java